client/src/components/search/SearchBox.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+
+export default function SearchBox() {
+  const [searchString, setSearchString] = useState("");
+
+  const loadSearchResults = () => {
+    window.location.href = "/search?username=" + searchString;
```

Comment:
Love the use of a route to load the page

Reply:
Thanks.